### PR TITLE
Scaffold online multiplayer

### PR DESCRIPTION
**Pull request by sebas42069**

### CODE REVIEW

File: Redoublet-website/src/components/CreateGameTable.vue [hunk 1]
```diff
@@ -0,0 +1,70 @@
+<script setup lang="ts">
+import type { Text } from 'vue';
+
+</script>
```
Comment (rubenfranquinet) [inline Redoublet-website/src/components/CreateGameTable.vue:4]: Compositions :)

File: Redoublet-website/src/views/MultiplayerCompetitionView.vue [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    </router-link>
+    <div class="row">
+        <div class="column">
+            <router-link to="/multiplayer-competitions/join" custom v-slot="{ navigate }">
+                <div 
+                class="btn"
+                @click="navigate"
+                >
+                join existing
+                </div>
```
Comment (rubenfranquinet) [inline Redoublet-website/src/views/MultiplayerCompetitionView.vue:17]: Also `<button>`

File: Redoublet-website/src/views/MultiplayerCompetitionView.vue [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+            </router-link>
+            <router-link to="/multiplayer-competitions/setup" custom v-slot="{ navigate }">
+                <div 
+                class="btn"
+                @click="navigate"
+                >
+                new online
+                </div>
```
Comment (rubenfranquinet) [inline Redoublet-website/src/views/MultiplayerCompetitionView.vue:25]: button please :)

File: Redoublet-website/src/views/MultiplayerTournamentView.vue [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    </router-link>
+    <div class="row">
+        <div class="column">
+            <router-link to="/multiplayer-tournaments/join" custom v-slot="{ navigate }">
+                <div 
+                class="btn"
+                @click="navigate"
+                >
+                Join existing tournament
+                </div>
```
Comment (rubenfranquinet) [inline Redoublet-website/src/views/MultiplayerTournamentView.vue:17]: Again `<button>`

File: Redoublet-website/src/views/MultiplayerTournamentView.vue [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+            </router-link>
+            <router-link to="/multiplayer-tournaments/setup" custom v-slot="{ navigate }">
+                <div 
+                class="btn"
+                @click="navigate"
+                >
+                Setup new online tournament
+                </div>
```
Comment (rubenfranquinet) [inline Redoublet-website/src/views/MultiplayerTournamentView.vue:25]: Almost there `<button>`

**Review by rubenfranquinet — CHANGES_REQUESTED** [review]
Please also change components name CreateGameTable.vue, JoinGameTable.vue and MenuBoxes.vue to be more intuitive

**Review by rubenfranquinet — APPROVED** [review]

**Comment by MatthijsdeHeus** [comment]
Maybe we can use v-for for the tables?

**Comment by rubenfranquinet** [comment]
> 

Will be done in next story, not in this one.

**Review by MatthijsdeHeus — APPROVED** [review]

**Pull request merged**